transcom/mymove
# MB-3243 Integrate Orders document viewer

## Description

This PR adds the ability for a TOO to view orders documents that the user uploaded.

## Reviewer Notes

Steps to test:
1. Run the setup commands in the section below
2. Visit http://officelocal:3000/devlocal-auth/login
2. Click the `Login` button next to `too_role@office.mil (PPM office)`
3. Click on the first `MoveOrderID` in the list
4. Change the url from http://officelocal:3000/moves/[move_id]/details to http://officelocal:3000/moves/[move_id]/orders

Expected Result: you should see a test.pdf file

By default, the seed data only uploads PDFs. To test other image types, follow these steps:

1. Sign in to http://milmovelocal:3000 as a service member
2. If you haven't created a move yet, go through onboarding and create one. When it's time to upload your orders documents, choose a PNG, JPG, or GIF. Please make sure they don't contain any PII. If you've already created a move, you can edit it and upload additional orders documents.
3. Visit http://officelocal:3000/devlocal-auth/login
4. Click the `Login` button next to `too_role@office.mil (PPM office)`
5. Click on the Move that you created as a service member.
6. Change the url from http://officelocal:3000/moves/[move_id]/details to http://officelocal:3000/moves/[move_id]/orders

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make client_run (in a separate terminal tab)
```

## Code Review Verification Steps

 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-3243) for this change

## Screenshots

<img width="1680" alt="Screen Shot 2020-09-10 at 2 59 48 PM" src="https://user-images.githubusercontent.com/811150/92786551-566e9680-f376-11ea-9ddd-539ac44d5bc3.png">
